Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -15,9 +15,57 @@
 
 @implementation BPPacker
 
++ (NSMutableSet *)fullyExpandedTests:(NSArray *)configTestCases fromTestFile:(BPXCTestFile *)xctFile {
+    NSMutableSet *expandedTests = [NSMutableSet new];
+    
+    for (NSString *testCase in configTestCases) {
+        if ([testCase rangeOfString:@"/"].location == NSNotFound) {
+            [xctFile.allTestCases enumerateObjectsUsingBlock:^(NSString *  _Nonnull actualTestCase, NSUInteger idx, BOOL * _Nonnull stop) {
+                if ([actualTestCase hasPrefix:[NSString stringWithFormat:@"%@/", testCase]]) {
+                    [expandedTests addObject:actualTestCase];
+                }
+            }];
+        } else {
+            [expandedTests addObject:testCase];
+        }
+    }
+    return expandedTests;
+}
+
++ (BPConfiguration *)normalizedBPConfiguration:(BPConfiguration *)config
+                                     testFiles:(NSArray *)xcTestFiles {
+    // config may include a list of tests to include or exclude.
```

Comment:
could you move this comments above the msg? Like this:

/*!
 * @discussion Pack a series of .xctest bundles into an NSArray of NSString with the testcases distributed according to the packing rules.
 * @param xcTestFiles An NSArray of BPXCTestFile's to pack
 * @param config The configuration file for this bluepill-runner
 * @return An NSMutableArray of BPBundle's with the tests packed into bundles.
 */